rdt-open-reader/src/main/java/io/ona/rdt/widget/RDTBarcodeFactory.java

```diff
@@ -90,13 +90,17 @@ private boolean isRDTExpired(Date date) {
     }
 
     protected void moveToNextStep(Date expDate) {
-        JsonFormFragment formFragment = widgetArgs.getFormFragment();
         if (!isRDTExpired(expDate)) {
             moveToNextStep();
         } else {
-            String expiredPageAddr = stepStateConfig.getStepStateObj().optString(PRODUCT_EXPIRED_PAGE, "step1");
-            JsonFormFragment nextFragment = RDTJsonFormFragment.getFormFragment(expiredPageAddr);
-            formFragment.transactThis(nextFragment);
+            navigateToUnusableProduct();
         }
     }
+
+    protected void navigateToUnusableProduct() {
```

Comment:
Please rename the method to `navigateToUnusableProductPage`.

Reply:
fixed.